Vagrantfile

```diff
@@ -9,6 +9,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.network :forwarded_port, guest: 8282, host: 8282
     config.vm.network :forwarded_port, guest: 80, host: 8000
     config.vm.network :forwarded_port, guest: 4443, host: 4443
+    config.vm.network :forwarded_port, guest: 777, host: 777
```

Comment:
yea hey I totally just tested it and it works, I will switch that up, that makes things way simpler.

looks like this:

```
  location /jsonrpc/ {
    if ($ssl_client_verify != "SUCCESS") {
      return 403;
    }
    proxy_pass          http://localhost:8282/jsonrpc/;
    proxy_read_timeout  90;
    proxy_redirect http://localhost/ $scheme://$host:$server_port/;
  }

  ssl_client_certificate /etc/nginx/jsonrpc-client.crt;
  ssl_verify_client optional;
```